packages/devextreme/testing/tests/DevExpress.ui.widgets/menu.tests.js

```diff
@@ -3502,6 +3502,34 @@ QUnit.module('adaptivity: behavior', {
         fx.off = false;
     }
 }, () => {
+    QUnit.test('Menu role updates correctly with adaptivity enabled on resize', function(assert) {
+        const menu = new Menu(this.$element, {
+            items: this.items,
+            adaptivityEnabled: true
+        });
+
+        $('#qunit-fixture').width('100%');
+        menu.repaint();
```

Comment:
I don't think we in tests should use repaint. I would test it in separate tests. Let's discuss this on our channel.

Reply:
[c317a55](https://github.com/DevExpress/DevExtreme/pull/28066/commits/c317a5579c7808781166a9bf40be333ba05ac031)